lib/ab/tests.rb

```diff
@@ -8,6 +8,14 @@ def before_picking_variant(&block)
       def after_picking_variant(&block)
         AssignedTest.after_picking_variant(&block)
       end
+
+      def logger=(logger)
+        @logger = logger
+      end
+
+      def logger
+        @logger ||= Logger.new($stdout)
```

Comment:
@redross imho, it should default to null instead of `$stdout`. Just like [event-tracker](https://github.com/vinted/event-tracker).